tests/pkg_micro-ecc/main.c

```diff
@@ -36,11 +36,39 @@
 
 #include <stdio.h>
 #include <string.h>
+#include "hashes/sha256.h"
 #include "uECC.h"
 #include "periph/hwrng.h"
 
 #define TESTROUNDS  16
 
+#ifndef FEATURE_PERIPH_HWRNG
```

Comment:
This part of the test could be run on all boards, even those with a rng.

Also, I would think it better, to have to seperate tests: One generic for all boards, and a seperate one for rng boards (like *tests/pkg_micro-ecc_withhwrng*). Otherwise, it would be possible to have a false positive test if for some reason the hwrng gets disabled at compile time.